packages/headless/src/controllers/facets/_common/facet-option-definitions.ts

```diff
@@ -15,15 +15,35 @@ import {FacetSearchRequestOptions} from '../../../features/facets/facet-search-s
 export const facetId = new StringValue();
 /** The field whose values you want to display in the facet.*/
 export const field = new StringValue({required: true});
+/**
+ * The base path shared by all values to display in the hierarchical facet.
+ * If you set this option, the specified base path will always be active on the facet.
+ */
 export const basePath = new ArrayValue({
   each: new StringValue(),
 });
-
+/**
+ * The character that specifies the hierarchical dependency.
+ * @default ','
```

Comment:
The default delimiting character is different for the facets (`>`) and category facets (`;`). With that said Kaitlyn is adding documentation in this same file.